src/core.js

```diff
@@ -69,6 +69,7 @@ const patchFactory = function(run) {
    * @param {(!Element|!DocumentFragment)} node
    * @param {!function(T)} fn
    * @param {T=} data
+   * @return {!Node} node
```

Comment:
Yep... I'm guessing Closure Compiler isn't actually checking this part because I didn't type `run` correctly.